rpc/container/rpcbinding.go

```diff
@@ -241,6 +241,11 @@ func New(actor Actor, hash util.Uint160) *Contract {
 	return &Contract{ContractReader{actor, hash}, actor, hash}
 }
 
+// Alias invokes `alias` method of contract.
+func (c *ContractReader) Alias(cid []byte) ([]byte, error) {
+	return unwrap.Bytes(c.invoker.Call(c.hash, "alias", cid))
```

Comment:
`unwrap.Bytes` won't be able to handle `Null` stackitem and will return an error. Is it the expected binding behaviour?

Reply:
It's expected. We can have `string` here, given the current state of `unwrap` that might be a bit more appropriate. But we couldn't have `nil, nil` return in the future this way. But the usefulness of `nil, nil` is questionable as well.

Reply:
Let's try strings.